doc/sphinx/develop/distribution-packages/pypi-sdist-wheel.md

```diff
@@ -130,8 +137,8 @@ built during the wheel build and must be pre-installed on the build system. I co
 find a way to have eigen and HighFive, respectively, find those dependencies if they
 were also installed using `FetchContent`. If Boost and libhdf5 are installed in
 non-standard directories, you can use the `Boost_ROOT` or `Boost_INCLUDE_DIRS` and
-`HDF5_ROOT` variables to specific the correct locations. See the CMake documentation for
-`find_package()`.
+`HDF5_ROOT` variables to specify the correct locations. See the CMake documentation for
+[`find_package()`](https://cmake.org/cmake/help/latest/command/find_package.html).
```

Comment:
The `Boost_ROOT` and `Boost_INCLUDE_DIRS` variables mentioned here should be all caps now, right?

Reply:
They can be either; `Boost` is the canonical spelling and I set up the configuration so that `BOOST` is also acceptable. `HDF5` is the canonical spelling of that library.